conf/application.conf

```diff
@@ -151,10 +142,10 @@ contact-frontend {
 }
 
 mongodb {
-  uri = "mongodb://localhost:27017/customs-declare-exports-frontend"
+  uri = "mongodb://localhost:27017/customs-movements-frontend"
 }
 
 urls {
   login = "http://localhost:9949/auth-login-stub/gg-sign-in"
-  loginContinue = "http://localhost:9000/customs-declare-exports-frontend"
+  loginContinue = "http://localhost:9000/customs-movements-frontend"
```

Comment:
Are you sure about port? I think we should use localhost with port for movements frontend

Reply:
you are right. missed that! will sort !